src/features/posts/postsSlice.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { Post } from '../../types/Post';
+import { getUserPosts } from '../../api/posts';
+
+type PostsInfo = {
+  items: Post[];
+  loaded: boolean;
+  hasError: boolean;
+};
+
+export const init = createAsyncThunk(
+  'userPosts/fetchPosts',
+  (userId: number) => {
+    return getUserPosts(userId);
+  },
+);
+
+const initialState: PostsInfo = {
+  items: [],
+  loaded: false,
+  hasError: false,
+};
+
+export const postsSlice = createSlice({
+  name: 'userPosts',
+  initialState,
+  reducers: {},
```

Comment:
It seems you are not using any reducer functions in your slice. If you don't need them you can omit the 'reducers' field.

Reply:
I tried and get an error - 'Property 'reducers' is missing in type...'. Also, 'reducers' generate corresponding action types and action creators, so, I assume, I should keep it in any case. If I'm wrong, please advise a solution.